main/main.go

```diff
@@ -158,3 +169,39 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func restoreIBU(ctx context.Context, client client.Client, log *logr.Logger) error {
+	ibu := &lcav1alpha1.ImageBasedUpgrade{}
+	filePath := filepath.Join(utils.Host, utils.IBUFilePath)
```

Comment:
```suggestion
	filePath := common.PathOutsideChroot(utils.IBUFilePath)
```
with: "github.com/openshift-kni/lifecycle-agent/internal/common"